WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <%= render layout: "nav" do %>
   <% cache ["registrations_index", @registrations.pluck(:id), @registrations.maximum(:updated_at)] do %>
-    <%= wca_table data: { toggle: "table", sort_name: "name" } do %>
```

Comment:
This invokes sorting on table load which is no longer appropriate. I've set up the arrow via JavaScript.